### DISCUSSION TITLE
For the people looking for the dotfiles in the README.md

### DISCUSSION THREAD

**Discussion by subterfugue** [williamboman/nvim-lsp-installer#559] [Show and tell]
it's from https://github.com/Avimitin/nvim :)

**Comment by williamboman** [comment]
Hehe if you're referring to the screenshots in the README, they're just screenshots of my personal setup.

- Colorscheme: https://github.com/rebelot/kanagawa.nvim
- Font: Jetbrains Mono Nerdfont with ligatures (with `adjust_line_height   5` & `adjust_column_width -1` [Kitty font settings])

**Reply by subterfugue** [reply]
Oh, I see! I guess I must have mis-identified it, sorry! Thank you very much for your help!
